quark.h

```diff
@@ -376,6 +376,8 @@ struct raw_shm {
 };
 
 struct quark_tty {
+	struct quark_tty *next;
+	size_t	total_len;	/* total bytes in the agg chain: next*/
 	u16	major;
 	u16	minor;
 	u16	cols;
```

Comment:
Can you align the whole structure, github doesn't let me edit all lines but here is what it should be:
```
	struct quark_tty	*next;
	size_t			 total_len;	/* total bytes in the agg chain: next */
	u16			 major;
	u16			 minor;
	u16			 cols;
	u16			 rows;
	u32			 cflag;
	u32			 iflag;
	u32			 lflag;
	u32			 oflag;
	size_t			 truncated;	/* how many bytes were truncated (lost) */
	size_t			 data_len;
	char			 data[];

```

Reply:
Pro tip, in the github profile -> settings -> appearance, there is a tab width preference, that for some ungodly reason is set to 4, I just found that now and my life is forever changed.